android/capacitor-fcm/src/main/res/layout/bridge_layout_main.xml

```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
why are you removing the bridge layout?

Reply:
This depends on `android.support.design.widget.CoordinatorLayout`, and I don't know if there's a way to dynamically switch between `android` and `androidx`. Is this even used in the package?

Reply:
I was thinking on publishing a v2 of this plugin using the "androidx way" for that. 

i.e.

```xml
<?xml version="1.0" encoding="utf-8"?>
<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    tools:context="com.getcapacitor.BridgeActivity"
    >

    <WebView
        android:id="@+id/webview"
        android:layout_width="fill_parent"
        android:layout_height="fill_parent" />

</androidx.coordinatorlayout.widget.CoordinatorLayout>

```

I got a capacitor-fcm v2 version working locally already. thoughts?

Reply:
That's definitely an option. However, is that even required? I was trying to keep it so the same version would be compatible with both versions of capacitor.

Reply:
makes sense. I'll check it tomorrow.